src/layout/GridList.jsx

```diff
@@ -28,8 +65,8 @@ export class GridListComponent extends React.Component {
 			'gridList',
 			{
 				[`gridList--has${columns.default}`]: !!columns.default,
-				[`atMedium_gridList--has${columns.medium}`]: !!columns.medium,
-				[`atLarge_gridList--has${columns.large}`]: !!columns.large,
+				[`atMedium_gridList--has${columns.medium || 0}`]: !!columns.medium,
```

Comment:
I'm a little confused here. How come 0 is needed when this should evaluate to `''` when the 0 would be added?

Reply:
I should come back and fix this, it was to appease flow and seemed to work at the time.